gfx/make_iso.sh

```diff
@@ -0,0 +1,76 @@
+# creates tileset_iso/ based on tileset/
```

Comment:
Just pick an interpreter at random. It's sure to work in at least one of them :)

(this script will be replaced with something in perl or python or another language that understands json natively, because it's going to need to do json modifications, including making new entries)